enterprise/backend/src/metabase_enterprise/sandbox/api/gtap.clj

```diff
@@ -20,11 +20,10 @@
     (db/select-one GroupTableAccessPolicy :group_id group_id :table_id table_id)
     (db/select GroupTableAccessPolicy {:order-by [[:id :asc]]})))
 
-#_{:clj-kondo/ignore [:deprecated-var]}
-(api/defendpoint-schema GET "/:id"
+(api/defendpoint GET "/:id"
```

Comment:
Totally unrelated, but I updated a handful of endpoints that weren't using any schemas. I couldn't resist